ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Mappings/TaskProfile.cs

```diff
@@ -13,8 +13,10 @@ public TaskProfile()
             CreateMap<Task, TaskDTO>();
             CreateMap<Status, StatusDTO>()
                 .ReverseMap();
-            CreateMap<CreateTaskDTO, Task>()
+            var map = CreateMap<CreateTaskDTO, Task>()
```

Comment:
don't need var map, you can write .ForMember just after first .ForMember but before `;`